Sources/RTVIClientIOS/RTVIClientVersion.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+extension RTVIClient {
+    
+    public static let libraryVersion = "0.2.0"
```

Comment:
We are currently publishing our libraries using SPM. This means the Git tag we create serves as the version number for our library. 
Unfortunately, SPM itself does not provide a built-in way to access this information at runtime from within the package’s code. That is why we are currently managing the versioning manually in the source code.

In the near future, we plan to improve this process by creating build scripts that will automatically update the version number and create the Git tag, ensuring they both use the exact same version.